pkg/metrics/features/metrics.go

```diff
@@ -512,6 +515,20 @@ func NewMetrics(withDefaults bool) Metrics {
 			Subsystem: subsystemACLB,
 			Name:      "cilium_node_config_enabled",
 		}),
+
+		NPL3Ingested: metric.NewGauge(metric.GaugeOpts{
```

Comment:
`NPL3Ingested` is defined as a `metric.Counter` but this is instantiated using `metric.NewGauge`. Not sure whether it matters.

Reply:
It's odd that it didn't complain, I'll change it to a counter.

Reply:
@tklauser it turns out that this should be defined as a metric.Gauge :sweat_smile: I forgot about this.

Reply:
> It's odd that it didn't complain, I'll change it to a counter.

I guess that's because the `metric.Counter` interfaceis a subset of the `metric.Gauge` interface.

> @tklauser it turns out that this should be defined as a metric.Gauge 😅 I forgot about this.

That's fine too. As long as they are consistent :sweat_smile: